Gemfile.lock

```diff
@@ -616,65 +618,63 @@ GEM
     sawyer (0.9.2)
       addressable (>= 2.3.5)
       faraday (>= 0.17.3, < 3)
-    sentry-rails (5.4.2)
+    sentry-rails (5.16.1)
       railties (>= 5.0)
-      sentry-ruby (~> 5.4.2)
-    sentry-ruby (5.4.2)
+      sentry-ruby (~> 5.16.1)
+    sentry-ruby (5.16.1)
       concurrent-ruby (~> 1.0, >= 1.0.2)
     sentry-user_informer (0.2.0)
       rack
       railties (>= 6.1.0, < 7.1.0)
       sentry-rails (~> 5.4)
-    sexp_processor (4.12.1)
-    single_cov (1.3.2)
-    slop (3.6.0)
+    sexp_processor (4.17.1)
+    single_cov (1.11.0)
     snaky_hash (2.0.1)
       hashie
       version_gem (~> 1.1, >= 1.1.1)
     socksify (1.7.1)
-    soft_deletion (1.6.0)
-      activerecord (>= 4.2.0, < 6.2.0)
+    soft_deletion (1.9.0)
+      activerecord (>= 5.0.0, < 7.2)
     sprockets (3.7.2)
       concurrent-ruby (~> 1.0)
       rack (> 1, < 3)
     sprockets-rails (3.4.2)
       actionpack (>= 5.2)
       activesupport (>= 5.2)
       sprockets (>= 3.0.0)
-    sqlite3 (1.4.2)
-    stackprof (0.2.12)
+    sqlite3 (1.7.2)
+      mini_portile2 (~> 2.8.0)
+    stackprof (0.2.26)
```

Comment:
### [stackprof](http://github.com/tmm1/stackprof)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 0.2.12 → 0.2.26

[_[change-log](https://github.com/tmm1/stackprof/blob/v0.2.26/CHANGELOG.md), [source-code](https://github.com/tmm1/stackprof/tree/v0.2.26), [gem-diff](https://my.diffend.io/gems/stackprof/0.2.12/0.2.26)_]





<details>
<summary>Commits</summary>

A change of **154** commits. See the full changes on [the compare page](https://github.com/tmm1/stackprof/compare/v0.2.12...v0.2.26).

These are the first 10 commits:
- (689cf7a) [Remove /ext from .gitignore](https://github.com/tmm1/stackprof/commit/689cf7ab3313f78557c8dfea1fdab40a733afa25)
- (4351979) [Merge pull request #99 from brixen/patch-1](https://github.com/tmm1/stackprof/commit/435197943428d790b8132110143dc084d91816a4)
- (01d3b08) [update gemfile](https://github.com/tmm1/stackprof/commit/01d3b08e8f0c5248534402ebb2d06d40811971aa)
- (b0fd99a) [Add ruby 2.5 to CI targets](https://github.com/tmm1/stackprof/commit/b0fd99abe2ee3a4b0784eaa17dff482c9dc8676c)
- (5b80393) [Merge pull request #101 from yui-knk/add_2_5](https://github.com/tmm1/stackprof/commit/5b803930d5b59a24fee8091643ed25825f6f95ed)
- (1af2220) [comment some of the inner workings](https://github.com/tmm1/stackprof/commit/1af222023e21b546b43df6f465672c53ad6504dd)
- (4e5ea0d) [feature: add --json format](https://github.com/tmm1/stackprof/commit/4e5ea0d27d4cd7937bcbbcf15339a3113f4482c0)
- (81da767) [Merge pull request #103 from palkan/feature/print-json](https://github.com/tmm1/stackprof/commit/81da767c1d7a9a898d99b750570338480f22baed)
- (43000fc) [Add test coverage around the string branch in result writing](https://github.com/tmm1/stackprof/commit/43000fca9add625f31dfeb4222297dfd151b26a4)
- (eba085e) [Flip conditional to use duck typing](https://github.com/tmm1/stackprof/commit/eba085e57c22de15b1d8cad20611f2dbece6ad65)


</details>